db/seedfiles/translations/en/me/shared.rb

```diff
@@ -41,6 +41,7 @@
     "en.has_eligible_health_coverage_from_hra" => "<b>Does this person currently have access to health coverage or a Health Reimbursement arrangement that they are not enrolled in (including through another person, like a spouse or parent)?</b> <br> <br> Select ‘yes’ if this person could get any of the types of health coverage listed below, either directly, or through another person such as a spouse or parent. If any of the types of health coverage listed below were available to this person, but they chose not to enroll, you should answer ‘yes’. <br> <ul> <li>Marketplace plan</li><li>MaineCare (Medicaid)</li><li>CubCare (Children’s Health Insurance Program)</li><li>Medicare</li><li>TRICARE</li><li>Coverage through a job (or another person's job, like a spouse or parent)</li><li>Health Reimbursement Arrangement (Individual Coverage HRA or Qualified Small Employer HRA)</li><li>COBRA</li><li>Retiree Health Benefits</li><li>Veterans Affairs (VA) Health Care Program</li><li>Peace Corps health benefits</li><li>Other full benefit coverage</li><li>Other limited benefit coverage</li></ul><p>A Health Reimbursement Arrangement is a type of group health plan that lets an employer reimburse a person for qualifying medical expenses, including their monthly health plan premium, in some cases. We need to know about any HRA that covers health insurance premiums that you have or could get from an employer.</p><p>To 'accept' an HRA offer, a person must tell the employer that they plan to use the HRA. If an HRA is available through an employer, but a person hasn't yet told the employer that they want to sign up for it, they aren't considered to be enrolled in an HRA. In some cases, a person may be able to change their mind about enrolling in an HRA they've signed up for, but hasn't started yet. If interested, ask the employer if this is an option.</p>",
     "en.has_eligible_health_coverage_from_hra_short" => "Does this person currently have access to health coverage or a Health Reimbursement Arrangement that they are not enrolled in?",
     "en.has_eligible_health_coverage_from_hra_short_min" => "Does this person currently have access to health coverage or a Health Reimbursement Arrangement that they are not enrolled in (including through another person, like a spouse or parent)?",
+    "en.has_eligibility_changed" => "",
```

Comment:
is this supposed to be empty?

Reply:
Lol... no. I've got multiple PR's with changes to this same file. This line will be updated in a different PR.